### DISCUSSION TITLE
Now playing doesn't change for livestreams

### DISCUSSION THREAD

**Discussion by IntoxicatedCookie** [AzuraCast/AzuraCast#4913] [Q&A]
Hello! My metadata doesn't change on the player when there's a new song on the livestream. It does show that there's a streamer live and plays the stream but the metadata won't update. It's stuck on the same metadata of the last song that the AutoDJ was playing before it gets interrupted by the incoming broadcast. I'm using Mixxx to stream. I've checked the option to dynamically update metadata in the Mixxx settings. What am I missing?

**Comment by BusterNeece** [comment]
This is a long-time known issue with Liquidsoap; I recently brought it up with them again on their GitHub, which you can follow here: savonet/liquidsoap#2109

**Reply by IntoxicatedCookie** [reply]
I see. Thank you, hope it gets resolved soon. One more thing, the metadata doesn't change on the Icecast administration panel either. Is that still due to Liquidsoap?

**Reply by IntoxicatedCookie** [reply]
> This is a long-time known issue with Liquidsoap; I recently brought it up with them again on their GitHub, which you can follow here: [savonet/liquidsoap#2109](https://github.com/savonet/liquidsoap/issues/2109)

I have been monitoring the situation and it seems like LiquidSoap has been updated to version 2.0.2 as well as Azuracast offering support for that version. I have updated my Azuracast installation and now the metadata *does* change but it still doesn't recognize the song. It either says "Live Broadcast" or "Unknown".
